src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+  width: 100px;
+  height: 116px;
+}
```

Comment:
You've fixed the size of `.page__body` which may lead to overflow issues if the content inside it grows. Unless there's a specific requirement, it's better to let the content dictate the size of its container.